extensions/ui/org.eclipse.smarthome.ui.classic/src/main/java/org/eclipse/smarthome/ui/classic/internal/servlet/WebAppServlet.java

```diff
@@ -275,7 +289,9 @@ public boolean hasChangeOccurred() {
          */
         @Override
         public void stateUpdated(Item item, State state) {
-            // ignore if the state did not change
+            if (item instanceof GroupItem) {
+                changed = true;

```

Comment:
We are in Home Automation systems, I doubt users have setup with lots of event updates per second. That is not my case.
I think I recently read that for standard groups there is no more any state update. Can you please confirm that point ? If that is true, the change will only concern groups with a function. If that is not true, I should change the code to trigger a reload on state update only for groups having a function.